<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="YCSLAP1132" failures="1" tests="1" name="Default test" time="28.589" errors="0" timestamp="2025-06-09T17:57:57 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="27.539">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at cogmentoCRM.Web.pageObjects.ContactsPage.validateCreatedContact(ContactsPage.java:66)
at cogmentoCRM.Web.stepDefinations.AddNewContact.validateCreatedContact(AddNewContact.java:77)
at âœ½.Contact should be added to the list in the Contacts page(file:///D:/Automation_Redefined/CogmentocrmWeb_BDD/src/test/resources/Features/AddNewContact.feature:12)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
